docs/cli/cli.md

```diff
@@ -0,0 +1,122 @@
+# 💻 CLI for `jstz`
+
+This guide will instruct through how to use the command line interface for `jstz`.
+
+## Overview
+
+`jstz` provides a simple command line interface where the user can easily run the sandbox environment and test smart functions.
+
+## Basic Commands
+
+- `jstz <COMMAND>`: The primary way to interact with `jstz`.
+- `help`: Displays a general help message or help information for specific subcommands.
+- `--version (-V)`: Displays the current version of `jstz`.
+
+## Sandbox
+
+The sandbox commands are responsible for managing the `jstz` sandbox environment.
+
+### Usage:
+
+```bash
+jstz sandbox <COMMAND>
+```
+
+### Commands:
+
+- `start`: Starts the sandbox environment.
+
+- `stop`: Shuts down the sandbox environment.
+
+## Bridge
+
+Bridge commands facilitate the interaction between L1 and L2.
+
+### Usage:
+
+```bash
+jstz bridge <COMMAND>
+```
+
+### Commands:
+
+- `deposit`: Transfer assets between an L1 sandbox address and an L2 sandbox address.
+
+### Usage:
+
+```bash
+jstz bridge deposit --from <FROM> --to <TO> --amount <AMOUNT>
+```
+
+### Options:
+
+- `--from (-f) <FROM>`: The L1 sandbox address or alias to withdraw from.
+
+- `--to (-t) <TO>`: The L2 sandbox address or alias to deposit to.
+
+- `--amount (-a) <AMOUNT>`: The quantity in ctez to transfer.
+
+### Example:
+
+```bash
+$ jstz bridge deposit --from tz4TGu1Bv6G5nbbQT6zCeMdWL2wFBJMu2Kv --to tz4N7y3T2e2dfCyHB1Ama68jnt3Fps7Ufu6d --amount 57
+
+
+```
+
```

Comment:
Would be good to show the output (if any). Additionally, instead of a section I think an example would look best in a custom container